mappings/net/minecraft/text/Text.mapping

```diff
@@ -95,6 +95,10 @@ CLASS net/minecraft/class_2561 net/minecraft/text/Text
 			ARG 1 type
 			ARG 2 context
 			ARG 3 json
+		METHOD method_43474 convertLiteralToString (Ljava/lang/Object;)Ljava/lang/Object;
```

Comment:
Maybe `optimizeArgument`? This method is applied to translation arguments to remove unnecessary indirections.